src/components/dialogs/create-study-dialog/upload-case.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Button from '@mui/material/Button';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useEffect } from 'react';
+import CircularProgress from '@mui/material/CircularProgress';
+
+const MAX_FILE_SIZE_IN_MO = 100;
+const MAX_FILE_SIZE_IN_BYTES = MAX_FILE_SIZE_IN_MO * 1024 * 1024;
+export const UploadCase = ({
```

Comment:
Where is it used ?

Reply:
yes, it is no longer used. We are using upload-new-case now. thank you